extensions/blocks/markdown/index.js

```diff
@@ -67,4 +67,14 @@ export const settings = {
 	edit,
 
 	save,
+
+	example: {
+		attributes: {
+			source:
+				'## ## Try Markdown\n\n' +
```

Comment:
What do you think about translating this text? It won't mean much for non-english speakers.

Reply:
Ah! I'll wrap in translations.

Reply:
How would I go about wrapping this in translations since it's multiple lines?

Reply:
How about something like this?

```diff
diff --git a/extensions/blocks/markdown/index.js b/extensions/blocks/markdown/index.js
index ad8cb53e6..75fdbfb61 100644
--- a/extensions/blocks/markdown/index.js
+++ b/extensions/blocks/markdown/index.js
@@ -14,6 +14,12 @@ import save from './save';
 
 export const name = 'markdown';
 
+const exampleTitle = __( 'Try Markdown', 'jetpack' );
+const exampleDescription = __(
+	'Markdown is a text formatting syntax that is converted into HTML. You can _emphasize_ text or **make it strong** with just a few characters.',
+	'jetpack'
+);
+
 export const settings = {
 	title: __( 'Markdown', 'jetpack' ),
 
@@ -70,11 +76,7 @@ export const settings = {
 
 	example: {
 		attributes: {
-			source:
-				'## ## Try Markdown\n\n' +
-				'Markdown is a text formatting syntax that is converted into HTML. ' +
-				'You can _emphasize_ text or **make it strong** with just a few characters.\n\n' +
-				'Search up "Markdown" to learn more.',
+			source: `## ## ${ exampleTitle }\n\n${ exampleDescription }`,
 		},
 	},
 };

```

Note that I removed "Search up "Markdown" to learn more." from the string since I didn't find it very useful.

Reply:
I'll try it out. Thanks!